lib/feeds/base.php

```diff
@@ -143,6 +143,22 @@ function get_xml_podcast_funding_node($url, $label)
     return $doc->saveXML($node);
 }
 
+function get_xml_podcast_soundbite_node($start, $duration, $title)
```

Comment:
Bitte den code von `lib/feeds/base.php` ins Modul ziehen. Kannst den tag über den hook `podlove_append_to_feed_entry` in den Feed ausgeben.